src/microservices/users/src/Api/Startup.cs

```diff
@@ -65,8 +53,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
                 app.UseDeveloperExceptionPage();
             }
 
+            app.UseHttpsRedirection();
```

Comment:
Not sure if we need this. Do we?

Reply:
Will remove it for now. This needs to be discussed on our next meeting.